source/common/modules/markdown-editor/table-editor/subview.ts

```diff
@@ -38,6 +38,10 @@ const ensureBoundariesFilter = EditorState.transactionFilter.of((tr) => {
     return tr // Do not mess with synchronizing transactions
   }
 
+  if (!tr.docChanged) {
```

Comment:
Ahhh, probably difficult. This transaction filter should ALSO disallow MOVEMENTS that go outside the subview boundaries. Unless you have a good reason for this change, I'd advise against it.